bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -54,3 +54,9 @@ admission:
         - {{.}}{{end}}{{range .ClusterCIDR}}
         - {{.}}{{end}}
     {{end}}
+apiServerArguments:
+  feature-gates:
+    - "ExperimentalCriticalPodAnnotation=true"
+    - "RotateKubeletServerCertificate=true"
+    - "SupportPodPidsLimit=true"
+    - "LocalStorageCapacityIsolation=false"
```

Comment:
@sjenning can you take a look? Every time we touch feature gates in the observers, we have to do the same here. Otherwise during bootstrapping the gates are wrong.

Reply:
does the apiserver care anything about these feature gates?

Reply:
as @sttts, we would need to keep these in sync with https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L57-L90

Reply:
@sjenning this is necessary for all operators where the operand uses those gates. And yes @ sync. We (as the non-owners of the features) don‘t really know where they influence which operand. The goal of the unit test is to make those own the bootstrap gates who already own the gates during normal operation.

Reply:
@sjenning there is a bz where we suspect that during the bootstrap minutes objects were created which failed to work later.